src/p_user.c

```diff
@@ -388,6 +388,8 @@ void P_PlayerThink (player_t* player)
       player->lookdir = 0;
       player->centering = false;
     }
+
+    player->slope = PLAYER_SLOPE(player);
```

Comment:
It is still set (a second time) in `P_MovePlayer()`, though.

Reply:
I would rather do the extra calculation (how many people actually use freelook?) than risk bugs in the future.